api/v1beta2/network_types.go

```diff
@@ -430,6 +439,26 @@ type SubnetSpec struct {
 
 	// Tags is a collection of tags describing the resource.
 	Tags Tags `json:"tags,omitempty"`
+
+	// ZoneType defines a zone type for this subnet.
+	//
+	// The valid values are availability-zone, and local-zone.
+	//
+	// Zone types local-zone is not selected to automatically create
+	// resources like Nat Gateway, Network Load Balancers, control plane or compute nodes.
+	//
+	// When local-zone, the public subnets will be associated with the public route table,
```

Comment:
When the `ZoneType` is `availability-zone`, what happens with the route table and subnets? They're just treated like normal ones in an AZ? Would be nice to document that explicitly.

Reply:
Yes, tables and subnets for regular zones (`availability-zone`) is not affected, keeping the regular flow. Thanks for the feedback, I will update the field documentation.